data/templates/dhcp-client/daemon-options.tmpl

```diff
@@ -1,4 +1,4 @@
 ### Autogenerated by interface.py ###
 
-DHCLIENT_OPTS="-nw -cf /var/lib/dhcp/dhclient_{{ifname}}.conf -pf /var/lib/dhcp/dhclient_{{ifname}}.pid -lf /var/lib/dhcp/dhclient_{{ifname}}.leases {{ifname}}"
+DHCLIENT_OPTS="-nw -cf /var/lib/dhcp/dhclient_{{ifname}}.conf -pf /var/lib/dhcp/dhclient_{{ifname}}.pid -lf /var/lib/dhcp/dhclient_{{ifname}}.leases{{" -e IF_METRIC=" ~ dhcp_options.default_route_distance if dhcp_options.default_route_distance is defined and dhcp_options.default_route_distance is not none}} {{ifname}}"
```

Comment:
What does the `~` do? I thought it's `+` to append a string.

Reply:
https://jinja.palletsprojects.com/en/2.10.x/templates/?highlight=operator#math

`~` is concatenation, `+` is more or less "Add or concat if add makes no sense".